packages/lab/src/__tests__/__e2e__/overlay/Overlay.cy.tsx

```diff
@@ -112,4 +115,26 @@ describe("GIVEN an Overlay", () => {
       });
     });
   });
+
+  describe("WHEN controlled", () => {
```

Comment:
tests make sure onOpenChange is called correctly when overlay is in controlled state

Reply:
This should also test it's called in an uncontrolled state

Reply:
added uncontrolled testing to test above L34